homeassistant/components/homeassistant/strings.json

```diff
@@ -27,6 +27,17 @@
     "no_platform_setup": {
       "title": "Unused YAML configuration for the {platform} integration",
       "description": "It's not possible to configure {platform} {domain} by adding `{platform_key}` to the {domain} configuration. Please check the documentation for more information on how to set up this integration.\n\nTo resolve this:\n1. Remove `{platform_key}` occurences from the `{domain}:` configuration in your YAML configuration file.\n2. Restart Home Assistant.\n\nExample that should be removed:\n{yaml_example}\n"
+    },
+    "storage_corruption": {
+      "title": "Storage corruption detected for `{file_key}`",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "[%key:component::homeassistant::issues::storage_corruption::title%]",
+            "description": "The `{file_key}` storage is corrupt.\n\nIf you made manual edits to the storage file, fix any syntax errors in `{corrupt_path}`, restore the file to the original path `{original_path}`, and restart. Otherwise, restore the system from a backup.\n\nClick SUBMIT below to confirm you have repaired or restored the file from a backup.\n\nThe exact error was: {error}"
```

Comment:
Because of our complicated restart many just saying "restart" is a bit ambiguous, it's better to be clear, "restart Home Assistant".

The instructions suggest to either fix the file or restore the system from a backup. However, the last sentence says `confirm you have repaired or restored the *file* from a backup`

Reply:
2a82a79fbf2e1a6285879ba3a4671ac066897a2a